### PR TITLE
Revert breaking change to peerDependencies

### LINKED ISSUE #62: Latest patch 1.1.1 introduces a breaking change and runtime error when running mocha tests
Running mocha tests with latest mocha-multi reporter plugin leads to breaking change and failed run. 
The following error is logged in the console and the run fails.  
```
runner.stats = stats;   
^ TypeError: Cannot set property stats of #<EventEmitter> which has only a getter
```

The error is not thrown in `"mocha-multi": "1.1.0"`.

### PR DESCRIPTION
**Pull request by mikeharder**
- Reverts a breaking change to peerDependencies introduced in #61
- Adds test coverage for mocha v2, v3, and v4
- Fixes #62

### CODE REVIEW

File: verify [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-mocha_multi="./mocha-multi.js"
+mocha_multi="mocha-multi.js"
```
Comment (mikeharder) [inline verify]: Require for tests to pass with `mocha` v2, v3, and v4.  I didn't investigate why this change is required on older versions of `mocha`, since it seems safe enough.  Also works for v5 and v6.

**Comment by glenjamin** [comment]
Published as `1.1.3`
